packages/ui/components/pages/Fuse/FusePoolEditPage/AssetConfiguration/AddAssetModal.tsx

```diff
@@ -135,7 +135,7 @@ const AddAsset = ({ comptrollerAddress, onSuccess, poolID, poolName }: AddAssetP
               poolData.assets.map((asset, index) => {
                 return (
                   <WrapItem key={index}>
-                    <Button variant="solid" px={2}>
+                    <Button variant="_solid" px={2}>
```

Comment:
Can't we use `solid` or adapt it? 
If we need to create a new style I personally would prefer a name without `_` underscore as it's indicating some variables to not mess/placeholder with instead of a `style` I should use all over the place

Reply:
Well, chakra provides some default variants buttons and I need them to tag, if we use 'solid` as before, then its style would be overrided

Reply:
Ah I see you changed all the variant names. But why again? It's a bit unclear to me.
Shouldn't using `solid` as before work?

<img width="1168" alt="Screenshot 2022-09-21 at 11 35 25" src="https://user-images.githubusercontent.com/839848/191470531-9b0cab4a-d4f6-4a04-bbea-a2b096a39c56.png">

Reply:
I am using default `solid` variant buttons as filters like `Collateral`, `Protected`, `Borrowable`. and if we use as before, then `solid` variants are overrided with our style so couldn't use it